src/visualizations/App.fs

```diff
@@ -136,6 +145,19 @@ let render (state: State) (_: Msg -> unit) =
                     | Failure error -> Utils.renderErrorLoading error
                     | Success data -> lazyView Map.mapChart {| mapToDisplay = Map.MapToDisplay.Region; data = data |} }
 
+    let skopjeMunMap =
+          { VisualizationType = SkopjeMunMap
+            ClassName = "sk-map-chart"
+            ChartTextsGroup = "sk-map"
+            Explicit = false
+            Renderer =
+                fun state ->
+                    match state.RegionsData with
```

Comment:
@lukarenko I would need your project knowledge and F# expertise here. If you look above at line 78-84 the variables `state.RegionsData` and `state.SkopjeMunicipalitiesData` contain the same data BUT if I pass `state.SkopjeMunicipalitiesData` as a variable here, the map does not work.

I am assuming it is something with the loading commands but I can't find out why. I searched all the usages of the Regions commands and `state.RegionsData` and replicated everything for the SkopjeMun use case but still I can't get it working.

Any help would be appreciated.